src/components/my-journey/CheckBox.tsx

```diff
@@ -1,3 +1,4 @@
+
 import React from "react";
 import { BiCheck } from "react-icons/bi";
```

Comment:
Line 17: instead of using a bunch of ternary operators, use the classnames library or clsx library

Reply:
`line 6 `and `line 7` please change the variable names to isStepper and isActive, respectively, as you did for isComplete, since they are boolean variables.